openapi/components/schemas/Organizations/OrganizationSettings.yaml

```
@@ -19,6 +19,17 @@ properties:
         nullable: true
         format: float
         description: Rate for flat tax calculation.
+  billing:
+    description: Global organization settings for billing.
+    type: object
+    nullable: true
+    properties:
+      pendingOrderTtl:
+        description: Time period in ISO-8601 durations format before pending order is automatically voided.
```

suggestion: I think it would be good to break this up a bit with 
```suggestion
        description: Length of time, in ISO-8601 durations format, before which a pending order is automatically voided.
```